source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -180,5 +181,5 @@ as shown in the following code:
    .. output::
       :language: none
 
-      NetworkDeviceRecord[deviceId=fedcba09876543210abcdef0, name=Enterprise Wi-fi, deviceType=router]
+      NetworkDeviceRecord[deviceId=fedc..., name=Enterprise Wi-fi, deviceType=router]
```

Comment:
Why the `...`?

Reply:
Unless we need to demonstrate the exact value, we use ellipses to truncate long strings that are part of sample or auto-generated data. We also use it to mark code that would depend on the user's setup.